timelock-agent/src/main/java/com/palantir/timelock/config/PaxosInstallConfiguration.java

```diff
@@ -27,8 +27,31 @@ default File dataDirectory() {
         return new File("var/data/paxos");
     }
 
+    /**
+     * Set to true if this is a new stack. Otherwise, set to false.
+     */
+    @JsonProperty("is-new-service")
+    boolean isNewService();
```

Comment:
what would be the default value here?

Reply:
There is no default value, and we'll throw an ISE if we try and build an object without it.

This is probably reasonable - in internal distributions we can specify it explicitly (probably `false`), and users can override it as needed using standard internal protocols.